GoogleForCreators/web-stories-wp
# Redesign: Separate textBox from textStyle panel.

## Context



## Summary
Splits up Text box and Text panels from TextStyle panel.


## Relevant Technical Choices



## To-do



## User-facing changes
| Before | After |
|--------|------|
| <img width="284" alt="Screenshot 2021-02-24 at 14 49 48" src="https://user-images.githubusercontent.com/3294597/109003008-32460e00-76a7-11eb-9ab6-0f587f272285.png"> | <img width="276" alt="Screenshot 2021-02-24 at 14 48 39" src="https://user-images.githubusercontent.com/3294597/109002880-117db880-76a7-11eb-8285-c0765040ea34.png"> |


## Testing Instructions

1. Add a text element and verify that it has Text box and Text style panels separated.


### QA
No need to do QA on this one -- QA will be done when actually redesigning these panels.


## Checklist




---



Fixes #6472
